packages/scandipwa/src/style/abstract/_parts.scss

```diff
@@ -31,3 +31,15 @@
         }
     }
 }
+
+@mixin description-tag-line-height {
+    li,
+    span,
+    div,
+    p,
+    dt,
+    dd,
+    dl {
+        line-height: 20px;
+    }
+}
```

Comment:
Better to add mixing with content, so that when you include it some where you add properties for it and it appliead to all those tags. In this case rename also mixing name.